psychopy/tools/filetools.py

```diff
@@ -241,3 +241,22 @@ def __del__(self):
             self.save()
         self._deleted = True
 
+def path_to_string(filepath):
```

Comment:
The function name should probably be in `mixedCase`, not `snake_case`, to fit in with other parts of PsychoPy.

Reply:
I'll add in another commit changing snake_case to camelCase, may not be able to do it tonight- but I'll have time to work locally tomorrow.